### DISCUSSION TITLE
Project Helps

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#173] [General]
I added a new file to the class-repo called Project Helps. We should add to it!

I added a question within the new file. If someone wants to add an answer there, I think it might help others with similar issues.

**Comment by CJ3315** [comment]
This is awesome! Thank you for adding this!

**Comment by kassyoveson** [comment]
Thanks, Skye, for the videos. It is nice to be reminded to keep things organized and not make silly mistakes such as spaces or long descriptions.
